dev/local-environment/configurations/genesis/shelley/genesis.json

```diff
@@ -1,23 +1,13 @@
 {
-  "activeSlotsCoeff": 0.8,
+  "activeSlotsCoeff": 0.4,
```

Comment:
Fixing discrepancy, the PC nodes are configured with 0.4. This is block per 2.5 seconds on average.